apis/feedapi/feedapi.yaml

```diff
@@ -0,0 +1,122 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: User Feed  API(s)
+  description: >-
+    - The User Feed API will notify user on various alerts like migration.
+
+    <b>Note:</b> These resources can be used as a part of scripts or programs to allow authorized administrators to manage the users.
+host: staging.open-sunbird.org
+basePath: /api/
+securityDefinitions:
+  bearer:
+    type: apiKey  # arbitrary name for the security scheme
+    in: header # The secrity scheme parameter is in "header"
+    name: Authorization # name of the header, query parameter or cookie
+  userToken:
+    type: apiKey  # arbitrary name for the security scheme
+    in: header # The secrity scheme parameter is in "header"
+    name: x-authenticated-user-token # name of the header, query parameter or cookie
+schemes:
+  - https
+paths:
+  /user/v1/feed/{User_ID}:
+    get:
+      description: >-
+        This API is used for searching a specific user, the user search is based on the unique ID of the user
```

Comment:
`Gets all the pending feeds for the user identified by userid`.
Remove the `The endpoint ...` line.

Reply:
done